src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -127,7 +126,8 @@ public static NewGameChooserEntry chooseGame(final Frame parent, final String de
   }
 
   private void selectGame(final String gameName) {
-    if (gameName == null) {
+    if (gameName == null || gameName.equals("-")) {
+      m_gameList.setSelectedIndex(0);//For some reason this is not working like it should...
```

Comment:
The problem is, that I wanted it to auto -select  the first entry, but for whatever reason this won't happen...